commands/board/list.go

```diff
@@ -142,7 +142,7 @@ func identify(pm *packagemanager.PackageManager, port *discovery.Port) ([]*rpc.B
 			logrus.Debug("Board not recognized")
 		} else if err != nil {
 			// this is bad, bail out
-			return nil, errors.Wrap(err, "error getting board info from Arduino Cloud")
+			return nil, errors.Wrap(err, tr("error getting board info from Arduino Cloud"))
```

Comment:
Should `Arduino Cloud` be translated? 🤔

Reply:
My guess is no, but this might be one of those cases where the loss of context in "error getting board info from %s" makes it more difficult for the translator to interpret the meaning?

I'm not really sure. I am not sufficiently experienced in this area to be able to make judgements at this level.